dev-docs/bidders/playdigo.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: bidder
+title: Playdigo
+description: Prebid Playdigo Bidder Adapter
+biddercode: playdigo
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+floors_supported: true
+gpp_supported: true
```

Comment:
Sorry - gpp_supported is no longer a valid piece of metadata. It's been replaced with gpp_sids as described in https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

Reply:
thanks for your reply, we have made changes